templates/tips/detail.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+Posted by {{tip.author}} at {{tip.date}} in {{ tip.category }}
```

Comment:
From this line I understand the date is for the date the Tip was first published
If we want to add the option to edit of a Tip, maybe it would be nice to add the last_edited date
What do you think? @Yoavsc129

Reply:
This page (tips/detail) is not used in the latest update and I might delete it

Reply:
deleted